docs/build/_index.md

```diff
@@ -1,9 +1,164 @@
+description: "Configure and program your smart machine with Viam."
+image: "/services/icons/sdk.svg"
+imageAlt: "Tools and building blocks."
 weight: 410
 type: "docs"
-layout: "empty"
-empty_node: true
+no_list: true
 menuindent: true
 ---
+
+Viam uses a composable system of building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} (components and services) so you can add the hardware drivers and the high-level software functionality you need to your smart machine:
+
+<div class="cards max-page">
+  <div class="row">
+    <div class="col sectionlist">
+        <div>
+        <h2>Components:</h2>
+        {{<sectionlist section="/components/">}}
+        </div>
+    </div>
+    <div class="col sectionlist">
+<div><h2>Services:</h2><ul class="sectionlist"><li><a href="/data/" title="Data Management Service"><div><picture><img src="/services/icons/data-capture.svg" width="/services/icons/data-capture" height="/services/icons/data-capture" alt="Data Management" loading="lazy"></picture><p>Data Management</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/motion/" title="Motion Service"><div><picture><img src="/services/icons/motion.svg" width="/services/icons/motion" height="/services/icons/motion" alt="Motion" loading="lazy"></picture><p>Motion</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/frame-system/" title="Robot Frame System"><div><picture><img src="/services/icons/frame-system.svg" width="/services/icons/frame-system" height="/services/icons/frame-system" alt="Frame System" loading="lazy"></picture><p>Frame System</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/base-rc/" title="Base Remote Control Service"><div><picture><img src="/services/icons/base-rc.svg" width="/services/icons/base-rc" height="/services/icons/base-rc" alt="Base Remote Control" loading="lazy"></picture><p>Base Remote Control</p></div></a></li></ul><ul class="sectionlist"><li><a href="/ml/" title="ML Model Service"><div><picture><img src="/services/icons/ml.svg" width="/services/icons/ml" height="/services/icons/ml" alt="ML Model" loading="lazy"></picture><p>ML Model</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/navigation/" title="Navigation Service"><div><picture><img src="/services/icons/navigation.svg" width="/services/icons/navigation" height="/services/icons/navigation" alt="Navigation" loading="lazy"></picture><p>Navigation</p></div></a></li></ul><ul class="sectionlist"><li><a href="/mobility/slam/" title="SLAM Service"><div><picture><img src="/services/icons/slam.svg" width="/services/icons/slam" height="/services/icons/slam" alt="SLAM" loading="lazy"></picture><p>SLAM</p></div></a></li></ul><ul class="sectionlist"><li><a href="/ml/vision/" title="Vision Service"><div><picture><img src="/services/icons/vision.svg" width="/services/icons/vision" height="/services/icons/vision" alt="Vision" loading="lazy"></picture><p>Vision</p></div></a></li></ul></div>
```

Comment:
Did you manually add a slash in front of each image src? I think that's the issue - I think it then cannot find the image
```suggestion
<div><div>Services</div><ul class="sectionlist"><li><a href="data/" title="Data Management Service"><div><picture><img src="services/icons/data-capture.svg" width="/services/icons/data-capture" height="/services/icons/data-capture" alt="Data Management" loading="lazy"></picture><p>Data Management</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/motion/" title="Motion Service"><div><picture><img src="services/icons/motion.svg" width="/services/icons/motion" height="/services/icons/motion" alt="Motion" loading="lazy"></picture><p>Motion</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/frame-system/" title="The Robot Frame System"><div><picture><img src="services/icons/frame-system.svg" width="/services/icons/frame-system" height="/services/icons/frame-system" alt="Frame System" loading="lazy"></picture><p>Frame System</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/base-rc/" title="Base Remote Control Service"><div><picture><img src="services/icons/base-rc.svg" width="/services/icons/base-rc" height="/services/icons/base-rc" alt="Base Remote Control" loading="lazy"></picture><p>Base Remote Control</p></div></a></li></ul><ul class="sectionlist"><li><a href="ml/" title="ML Model Service"><div><picture><img src="services/icons/ml.svg" width="/services/icons/ml" height="/services/icons/ml" alt="ML Model" loading="lazy"></picture><p>ML Model</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/navigation/" title="The Navigation Service"><div><picture><img src="services/icons/navigation.svg" width="/services/icons/navigation" height="/services/icons/navigation" alt="Navigation" loading="lazy"></picture><p>Navigation</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/slam/" title="SLAM Service"><div><picture><img src="services/icons/slam.svg" width="/services/icons/slam" height="/services/icons/slam" alt="SLAM" loading="lazy"></picture><p>SLAM</p></div></a></li></ul><ul class="sectionlist"><li><a href="ml/vision/" title="Vision Service"><div><picture><img src="services/icons/vision.svg" width="/services/icons/vision" height="/services/icons/vision" alt="Vision" loading="lazy"></picture><p>Vision</p></div></a></li></ul></div>
```

Reply:
I did because locally, it didn't work without the slash.

Reply:
Yeah my solution is wrong, this needs to use relative links which are different if they come from the build page:
```suggestion
<div><div>Services</div><ul class="sectionlist"><li><a href="data/" title="Data Management Service"><div><picture><img src="../services/icons/data-capture.svg"   alt="Data Management" loading="lazy"></picture><p>Data Management</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/motion/" title="Motion Service"><div><picture><img src="../services/icons/motion.svg" alt="Motion" loading="lazy"></picture><p>Motion</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/frame-system/" title="The Robot Frame System"><div><picture><img src="../services/icons/frame-system.svg" alt="Frame System" loading="lazy"></picture><p>Frame System</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/base-rc/" title="Base Remote Control Service"><div><picture><img src="../services/icons/base-rc.svg" alt="Base Remote Control" loading="lazy"></picture><p>Base Remote Control</p></div></a></li></ul><ul class="sectionlist"><li><a href="ml/" title="ML Model Service"><div><picture><img src="../services/icons/ml.svg" alt="ML Model" loading="lazy"></picture><p>ML Model</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/navigation/" title="The Navigation Service"><div><picture><img src="../services/icons/navigation.svg" alt="Navigation" loading="lazy"></picture><p>Navigation</p></div></a></li></ul><ul class="sectionlist"><li><a href="mobility/slam/" title="SLAM Service"><div><picture><img src="../services/icons/slam.svg" alt="SLAM" loading="lazy"></picture><p>SLAM</p></div></a></li></ul><ul class="sectionlist"><li><a href="ml/vision/" title="Vision Service"><div><picture><img src="../services/icons/vision.svg" alt="Vision" loading="lazy"></picture><p>Vision</p></div></a></li></ul></div>
```

Reply:
Ah and now the hrefs need to be absolute links but the relative icons work! cool